src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int NUMBER = 100;
```

Comment:
If we only use a constant inside a class, it should be private
```suggestion
    private static final int NUMBER = 100;
```